recipe/meta.yaml

```diff
@@ -6,7 +6,7 @@
 # of the C++ stdlib; keep the variable in case this happens again
 {% set libcxx_major = major_ver %}
 
-{% set build_number = 21 %}
+{% set build_number = 22 %}
```

Comment:
Main is already on 22
https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/d8ea0127a69042f122d94299c2948bdaafac1707/recipe/meta.yaml#L9
so this needs another bump